libs/components/src/lib/popup/popup.spec.ts

```diff
@@ -148,7 +148,7 @@ describe('vwc-popup', () => {
 
 		it('should fire open & close events', async function () {
```

Comment:
This tests only `open`. 
```suggestion
		it('should fire vwc-popup:open event', async function () {
```